app/controllers/project_database.py

```diff
@@ -369,8 +381,20 @@ def get(self, database_id):
 
         if errors:
             return dict(status='fail', message=errors), 500
+        
+        # Check the database status on host
+        database_service = DatabaseService()
+        database_connection = database_service.create_db_connection(user=database.user, password=database.password, db_name=database.name)
```

Comment:
@MutegekiHenry , please close this connection after you create it

Reply:
okay